src/Layout/index.jsx

```diff
@@ -1,16 +1,39 @@
-import React from 'react';
-import { data } from '../data';
-import { GtaVersion } from '../components/gtaVersions';
+import React, { useState, useEffect } from 'react';
+import { getAllVersions } from '../services';
+import { Button } from '../components/modal/button';
+import { AddVersionModal } from '../components/shared/addVersionModal/addVersionModal';
+import { GtaVersion } from '../components/gtaVersion';
 
 import './styles.scss';
 
 const Layout = () => {
+  const [versions, setVersions] = useState([]);
+  const [addVersionActive, setAddVersionActive] = useState(false);
```

Comment:
Make convinient namings. `updateModalVisible` and `addVersionActive` not much to same naming convention